app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_comment_counter
+    post.increment(:commentscounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please consider using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_comment_counter` method every time a new comment is created and saved to the database, then make the update_post_comment_countermethod private